mesh/model.c

```diff
@@ -884,14 +884,44 @@ static void send_dev_key_msg_rcvd(struct mesh_node *node, uint8_t ele_idx,
 					   size, data);
 }
 
+static inline bool match_opcode(const void *a, const void *b)
+{
+	return *(const uint32_t *)a == *(const uint32_t *)b;
```

Comment:
`L_PTR_TO_UINT`

Reply:
This is correct. `L_PTR_TO_UINT` is to extract `unsigned int` from pointer value.

```#define L_PTR_TO_UINT(p) ((unsigned int) ((uintptr_t) (p)))```

Reply:
Right, but see the other comment. Sorry I wasn't clear.